src/Components/Onboarding/Wizard.tsx

```diff
@@ -4,6 +4,7 @@ import { StepComponent } from "./Types"
 interface Props {
   stepComponents: StepComponent[]
   redirectTo?: string
+  forceStep?: string
```

Comment:
This may be a bit exhausting to maintain, but I'd like a more strict type check here so human errors like a typo could be caught earlier:

```ts
import Budget from './Steps/Budget'
...

interface Props {
  ...
  forceStep?: Budget.slug | ... // 
}
```

Reply:
👍 Oh, I like it!